### DISCUSSION TITLE
v1.2.27 released twice

### DISCUSSION THREAD

**Discussion by oh2fih** [kanboard/kanboard#5183] [General]
I noticed that v1.2.27.zip does not match the sha512sum recorded on Mon Mar 6 12:47:15 2023 +0200.

`b136d9c3ce1ea74cabc786ef13f9621c58094f7b165422fda86a96812c90ad605bfad14ee8e1036edc5f87c5a779c57edb62eeab5efc2231b63e29a9c2288c85  v1.2.27.zip`

It seems the v1.2.27 has been released twice? If there has been modifications or bug fixes it would be better to release a new version (v1.2.27.1 or v1.2.28) to avoid unnecessary ambiguity. Otherwise you could get non-repeatable bug reports from early birds.

**Comment by fguillot** [comment]
Version 1.2.27 was released only once. The Git tag was created only once.

These archives are generated automatically by GitHub.

It's possible that the problem is coming from this GitHub announcement: https://github.blog/changelog/2023-01-30-git-archive-checksums-may-change/

I get a different result than yours:

```bash
wget https://github.com/kanboard/kanboard/archive/refs/tags/v1.2.27.zip
sha256sum v1.2.27.zip                                                  
acc3328d19e45f93ef050f6c076676cc2cf4bebdee5c10aa87c81beb3e10bd44  v1.2.27.zip
```

**Comment by oh2fih** [comment]
Thanks for clarifying this. It's good they have reverted the change now.

That was a sha512sum, though. But I now get the same sha256sum:

```
$ sha256sum v1.2.27.zip
acc3328d19e45f93ef050f6c076676cc2cf4bebdee5c10aa87c81beb3e10bd44  v1.2.27.zip
$ sha512sum v1.2.27.zip
f6fff7167268a0226a238da19f8912326d77f6c2e4a94b3e577ab362a51f79261a062463afcca54dd5c06740090123567293cbcdd5ca4698bb47af8e879a95a1  v1.2.27.zip
```
